src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -281,7 +281,7 @@ export function* lcManager(params={}) {
 function updateRawTableChart(timeCName, fluxCName, converterId) {
 
     if (timeCName && fluxCName) {
-
+        const missionName = getConverter(converterId).missionName;
```

Comment:
suggestion: should we capitalize mission name so that `ztf` becomes `ZTF`, wise becomes `WISE`, and so on?